src/main/java/co/omise/models/SourceType.java

```diff
@@ -32,6 +32,8 @@
     InstKtc,
     @JsonProperty("installment_kbank")
     InstKBank,
+    @JsonProperty("installment_scb")
+    InstScb,
     @JsonEnumDefaultValue
     Unknown,
```

Comment:
We have to add new case to `toString` method too. e.g.

```java
    public String toString() {
        switch (this) {
            case InternetBankingBay:
                return "internet_banking_bay";
            // here
```

Reply:
@rezigned Added here: https://github.com/omise/omise-java/pull/116/commits/a4c18b463bfa55fbbacc4480071bff27f02620e2